SQL/Cleanup_patches/2018-02-20_remove_duplicate_value_from_Config.sql

```diff
@@ -0,0 +1,10 @@
+/*  delete duplicate ConfigID-value pairs */
+
+Delete from Config  Where ID in
+    (
+      select temp.ID from
+       (select ID from Config c
+          where (c.ConfigId,c.Value) in (select ConfigId,Value from Config group by ConfigId,Value having count(*) > 1)
+          and ID not in (select min(ID) from Config group by ConfigId,Value having count(*)>1)
```

Comment:
I don't think the logic of this is correct because of the number of sub-queries. The `MIN()` from the `ID NOT IN` part is referencing a different ID than the first part of the where clause. (At any rate, it's hard to follow enough to verify..)

Couldn't this be done with a single group by?

Reply:
@driusan The query is long and complex but it make sense to select all ID except the first one (lowest ID) in case of duplication.